File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroRES1001U00DoctorReservationStatusListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable ]
	public class NuroRES1001U00DoctorReservationStatusListItemInfo
	{
		private String _doctorName;
		private String _reservationStatus;

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String ReservationStatus
		{
			get { return this._reservationStatus; }
			set { this._reservationStatus = value; }
		}

		public NuroRES1001U00DoctorReservationStatusListItemInfo() { }

		public NuroRES1001U00DoctorReservationStatusListItemInfo(String doctorName, String reservationStatus)
		{
			this._doctorName = doctorName;
			this._reservationStatus = reservationStatus;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroRES1001U00DoctorReservationStatusListItemInfo.java
package nta.med.data.model.ihis.nuro;

public class NuroRES1001U00DoctorReservationStatusListItemInfo {
	private String doctorName;
	private String reservationStatus;
	
	public NuroRES1001U00DoctorReservationStatusListItemInfo(String doctorName,
			String reservationStatus) {
		this.doctorName = doctorName;
		this.reservationStatus = reservationStatus;
	}

	public String getDoctorName() {
		return doctorName;
	}

	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}

	public String getReservationStatus() {
		return reservationStatus;
	}

	public void setReservationStatus(String reservationStatus) {
		this.reservationStatus = reservationStatus;
	}
}
